script/tool/lib/src/publish_plugin_command.dart

```diff
@@ -63,13 +63,21 @@ class PublishPluginCommand extends PluginCommand {
       // Flutter convention is to use "upstream" for the single source of truth, and "origin" for personal forks.
       defaultsTo: 'upstream',
     );
+    argParser.addOption(
+      _allOption,
+      help:
+          'Based the diff between $_kBaseShaOption and HEAD, publish all the plugins that has pubspec change in this diff.\n'
+          'The --package option is ignored if this flag is on. (This is currently under construction)' ,
```

Comment:
What does "under construction" mean here? Is the plan for this behavior to change?

Reply:
It meant it is not supported yet. I had everything wired up in a different branch but i thought to introduce this pr as an intermediate step is easier to review. 

 I should just remove the flag int this PR as well and add this flag when actually adding this feature.